django/contrib/staticfiles/finders.py

```diff
@@ -121,9 +123,26 @@ def list(self, ignore_patterns):
         List all files in all locations.
         """
         for prefix, root in self.locations:
-            storage = self.storages[root]
-            for path in utils.get_files(storage, ignore_patterns):
-                yield path, storage
+            if os.path.isdir(root):
+                storage = self.storages[root]
+                for path in utils.get_files(storage, ignore_patterns):
+                    yield path, storage
+            else:
+                printer = MessagePrinter()
+                message = "Path [%s] does not exists. Skipping..." % str(root)
+                printer.log(message)
```

Comment:
`self.stderr.write(msg)` will throw an error here as 

> 'FileSystemFinder' object has no attribute 'stderr'

, `stderr` is defined under `BaseCommand`